jme3-core/src/main/java/com/jme3/anim/AnimComposer.java

```diff
@@ -526,7 +530,9 @@ public void cloneFields(Cloner cloner, Object original) {
         for (String key : layers.keySet()) {
             newLayers.put(key, cloner.clone(layers.get(key)));
         }
-
+        if (!newLayers.containsKey(DEFAULT_LAYER)) {
```

Comment:
Please use `map.putIfAbsent(key, value)` instead. This is generally considered more concise and potentially more efficient than the manual check.

eg:
```java
map.putIfAbsent(DEFAULT_LAYER, new AnimLayer(DEFAULT_LAYER, null);
```